pkg/users/account-create-dialog.js

```diff
@@ -287,9 +337,23 @@ export function account_create_dialog(accounts) {
                     caption: _("Create"),
                     style: "primary",
                     clicked: () => {
-                        return passwd_check(false, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.change_passw_force);
+                        return passwd_check(
+                            false,
+                            false,
```

Comment:
do you mind adding comments to the `false` values what they mean? same below for `true`. This helps to understand the code.